### PR TITLE
Integration test sample for .NET 7

### PR DESCRIPTION
**Pull request by jvandertil**
As requested by @Rick-Anderson in https://github.com/dotnet/AspNetCore.Docs/issues/27089

### CODE REVIEW

**Comment by jvandertil** [comment]
Currently away on vacation so a bit on and off on this PR. Tests are passing atleast. Will try to address the issues raised in the linked issue.

**Comment by Rick-Anderson** [comment]
@jvandertil much appreciated, no hurry.

**Comment by jvandertil** [comment]
@Rick-Anderson Not sure if this example covers all the cases you have in mind, but should be a nice starting point. Let me know if you want more things added, or changed, or if those should be in a separate PR.

**Comment by Rick-Anderson** [comment]
@jvandertil  can you force push `.sln`
See https://github.com/dotnet/AspNetCore.Docs/blob/main/.gitignore#L15-L17
this gitignore is a copy

File: test/integration-tests/IntegrationTestsSample/src/RazorPagesProject/Data/ApplicationDbContext.cs [hunk 1]
```diff
@@ -0,0 +1,77 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore;
+
+namespace RazorPagesProject.Data
```
Comment (Rick-Anderson) [inline test/integration-tests/IntegrationTestsSample/src/RazorPagesProject/Data/ApplicationDbContext.cs:7]: If you change all of the namespaces to [File scoped namespaces ](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-10.0/file-scoped-namespaces)
`namespace RazorPagesProject.Data`
VS will format the code for you.

**Review by Rick-Anderson — APPROVED** [review]

**Comment by Rick-Anderson** [comment]
@jvandertil  I can do a PR to add the SLN file and fix the namespaces if you like. Great work.

File: test/integration-tests/IntegrationTestsSample/src/RazorPagesProject/Data/ApplicationDbContext.cs (hunk 1 above)
Comment (jvandertil) [reply test/integration-tests/IntegrationTestsSample/src/RazorPagesProject/Data/ApplicationDbContext.cs]: Updated all .cs files to use file scoped namespaces. Also aligned the namespaces with the folder structure which was not consistent across files.

**Comment by jvandertil** [comment]
@Rick-Anderson Added solution file and fixed namespaces

**Comment by Rick-Anderson** [comment]
@jvandertil much appreciated.

**Pull request merged**